src/main/java/com/example/yatzysimulator/dto/ScoreOfCategories.java

```diff
@@ -0,0 +1,15 @@
+package com.example.yatzysimulator.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ScoreOfCategories {
```

Comment:
All DTOs should be named like SomethingDTO or SomethingRequest/SomethingResponse or so.
I recommend `CategoryScoreDto`